ansible/roles/storage/tasks/debian.yml

```diff
@@ -9,3 +10,16 @@
   retries: 5
   until: apt_install_iscsi is success
   notify: restart iscsid
+
+- name: copy iscsi_tcp config file
+  copy:
+    src: openebs/iscsi_tcp.conf
+    dest: '/etc/modules-load.d/iscsi_tcp.conf'
```

Comment:
From what I remember there's a few other modules that need to be loaded as well. Perhaps we should call the .d `openebs_iscsi`. I'm having trouble finding the other modules in their docs though in case any other iscsi modules need to be loaded in the future it might be better off to just use a generic name.

Reply:
I was following this documentation, https://github.com/openebs/cstor-operators/blob/master/docs/quick.md#prerequisites we can certainly rename this file though to be more intuitive, great suggestion!